packages/plugin-awesome/tsconfig.test.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
There is no need to add additional configuration for typescript there. Please, remove the file